packages/core/botpress-builtins/src/setup.js

```diff
@@ -41,8 +48,14 @@ export default function(bp) {
       return next()
     }
 
-    const position = await bp.dialogEngine.getCurrentPosition(stateId)
-    await bp.users.tag(stateId, USER_TAG_CONVO_LAST, position && position.flow)
+    // TODO: eliminate simultaneous INSERTs that violate
+    // unique constraint
+    try {
+      const position = await bp.dialogEngine.getCurrentPosition(stateId)
+      await bp.users.tag(stateId, USER_TAG_CONVO_LAST, position && position.flow)
+    } catch (err) {
+      // console.error(err.message)
```

Comment:
@emirotin , should it be commented out?

Reply:
I've left it intentionally as it was spamming a lot during stress testing. Will uncomment and let's see if it's annoying